### ISSUE TITLE
Improve image drop zone contrast ratio

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#2134]
Any text should have a color contrast ratio of at least 4.5:1 with its background. The current contrast for text and background of the image "dropzone" is too low, and I'd say it's a bit too low even for people without vision impairments:

![screen shot 2017-08-01 at 21 49 02](https://user-images.githubusercontent.com/1682452/28844298-70c20f18-7704-11e7-94cf-37f46d21345d.png)

the one in the classic editor is better (can't measure it exactly because it's semi-transparent):

![screen shot 2017-08-01 at 21 50 06](https://user-images.githubusercontent.com/1682452/28844327-8df14efa-7704-11e7-8b03-fbbc26443e7e.png)

Also the text size matters. Text bigger than 18.5 pixels and bold, or text bigger than 24 pixels, can have a contrast ratio of just 3:0.

**Comment by wtfaremyinitials** [comment]
I'll give this a shot

**Comment by youknowriad** [comment]
closing as already fixed by #2767
